site2/docs/admin-api-brokers.md

```diff
@@ -4,16 +4,26 @@ title: Managing Brokers
 sidebar_label: Brokers
 ---
 
+> **Important**
+>
+> This page only shows **some frequently used operations**.
+>
+> - For the latest and complete information about `pulsar-admin`, including commands, flags, descriptions, and more, see [here](http://pulsar.apache.org/tools/pulsar-admin/).
+> 
+> - For the latest and complete information about `REST API`, including parameters, responses, samples, and more, see [here]{@inject: rest:REST:/}.
```

Comment:
nit: no need for backticks here and on the below line, since we're not referring to a command. It can simply be "the REST API" and "the Java admin API".

Reply:
Since I want to emphasize the keyword (pulsar-admin, rest API, java admin api), so I use a special format (backticks).

Reply:
Typically you'd use *italic* or **bold** text for emphasis. Anyway you're still missing the article _the_ in front of "REST API" and "admin API"